stock_picking_backorder_strategy/models/stock_picking_type.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models, _
+
+
+class StockPickingType(models.Model):
+    _inherit = 'stock.picking.type'
+
+    backorder_strategy = fields.Selection(
+        [('manual', _('Manual')), ('create', _('Create')),
```

Comment:
Remove `_()` function, selection fields are translatable by default, so using it there makes no sense.

This applies for all selections. Since `_` won't be used, you'll have to remove it also the import statement.